lib/rspec/snapshot.rb

```diff
@@ -1,4 +1,6 @@
-require "rspec"
-require "rspec/snapshot/version"
-require "rspec/snapshot/configuration"
-require "rspec/snapshot/matchers"
+require 'rspec'
+require 'rspec/snapshot/version'
+require 'rspec/snapshot/configuration'
+require 'rspec/snapshot/utils'
```

Comment:
added `utils` and `load_snapshot`

TODO: revert the `"` to `'` changes

Reply:
yes, plz don't change `"`